requirements_oracle_ds.txt

```diff
@@ -1,4 +1,5 @@
-# Requires installation of, or similar versions of:
+# If you want to use thick mode, you need to install the Oracle client library.
+# reference. https://pypi.org/project/oracledb/#description
 #    oracle-instantclient12.2-basic_12.2.0.1.0-1_x86_64.rpm
 #    oracle-instantclient12.2-devel_12.2.0.1.0-1_x64_64.rpm
-cx_Oracle==5.3
+oracledb==1.4.0
```

Comment:
As this will work without needing the Oracle client pieces installed first, we should probably move this `oracledb=1.4.0` stuff into the [requirements_all_ds.txt](https://github.com/getredash/redash/blob/master/requirements_all_ds.txt) file instead. :smile:

Reply:
> we should probably move this oracledb=1.4.0 stuff into the [requirements_all_ds.txt](https://github.com/getredash/redash/blob/master/requirements_all_ds.txt) file instead. 😄

I agree.
However, I thought it would be a major change and would prefer another opportunity.

I will try to move it to requirements_all_ds.txt and test the dependencies.